common/src/seg_tree.c

```diff
@@ -72,19 +72,22 @@ void seg_tree_destroy(struct seg_tree* seg_tree)
     seg_tree_clear(seg_tree);
 };
 
-/* Allocate a range tree.  Free it with free() when finished */
+/* Allocate a node for the range tree.  Free node with free() when finished */
 static struct seg_tree_node*
 seg_tree_node_alloc(unsigned long start, unsigned long end, unsigned long ptr)
 {
+    /* allocate a new node structure */
     struct seg_tree_node* node;
     node = calloc(1, sizeof(*node));
     if (!node) {
         return NULL;
     }
 
+    /* record logical range and physical offset */
     node->start = start;
-    node->end = end;
-    node->ptr = ptr;
+    node->end   = end;
+    node->ptr   = ptr;
```

Comment:
Please leave this as it was.  I don't want to start a convention where we have to line up all our `=` signs.